nil-docs/zkllvm/getting-started/compiling-a-circuit.mdx

```diff
@@ -0,0 +1,111 @@
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+# Compiling a circuit
+
+This tutorial contains step-by-step instructions on how to compile the [**elliptic curve circuit created previously**](./compiling-a-circuit.mdx).
```

Comment:
This link should probably lead to "Writing a simple circuit", right?

Reply:
thank you, you are right